tasks/github_tasks.py

```diff
@@ -135,39 +137,98 @@ def lint_codeowner(_):
     root_folder = os.path.join(base, "..")
     os.chdir(root_folder)
 
-    owners = _get_code_owners(root_folder)
+    exit_code = 0
 
-    # make sure each root package has an owner
-    pkgs_without_owner = _find_packages_without_owner(owners, "pkg")
-    if len(pkgs_without_owner) > 0:
-        raise Exit(
-            f'The following packages  in `pkg` directory don\'t have an owner in CODEOWNERS: {pkgs_without_owner}',
-            code=1,
-        )
+    # Getting GitHub CODEOWNER file content
+    owners = read_owners(owners_file)
+
+    # Define linters
+    linters = [_has_packages_without_owner, _has_orphans_in_codeowner]
+
+    # Execute linters
+    for linter in linters:
+        if linter(owners):
+            exit_code = 1
 
+    raise Exit(code=exit_code)
+
+
+def _has_packages_without_owner(owners, folder="pkg"):
+    error = False
 
-def _find_packages_without_owner(owners, folder):
-    pkg_without_owners = []
     for x in os.listdir(folder):
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏Should you use https://github.com/DataDog/datadog-agent/blob/d4e9db1893d990e9a1688372dad955a2de41f468/tasks/modules.py#L130 ?

Reply:
It could have been a great idea but some packages in `pkg/` are not modules, so they are note referenced in DEFAULT_MODULE :/